src/views/3-Modeling/Modeling.vue

```diff
@@ -687,113 +691,114 @@
     },
     data() {
           return {
+            // pull title, text, and methods 
             text: modelingText.textContents,
+            sectionTitle: "Predicting water temperature", // the initial
+
             publicPath: process.env.BASE_URL, // this is need for the data files in the public folder, this allows the application to find the files when on different deployment roots
             d3: null, // this is used so that we can assign d3 plugins to the d3 instance
 
             // dimensions
-            margin: 20,
-            width: 1000,
             height: 1000,
-            marginX: 20,
-            marginY: 20,
+            width: 1000,
+            margin: 50,
+            svg: null,
+
+            chartState: {},
+            model_exp: {error: 'error_x', ANN: 'ANN', RNN: 'RNN', RGCN: 'RGCN', RGCN_ptrn: 'RGCN_ptrn'},
+            model_data: {long: 'rmse_monthly', cast:'rmse_monthly_cast'},
 
             // beeswarm
-            radius: 4,
+            radius: 8,
+            set_colors: null,
+            color_exp: null, 
+            paddedRadius: null,
             bees: null,
-            svg: null,
             xScale: null,
-            model_sel: null,
-            rmse_monthly: null,
-            rmse_monthly_cast: null,
+            yScale: null,
+            rmse_monthly: [],
+            rmse_monthly_cast: [],
+            simulation: null,
 
             // scroll options
             scroller: null,
-            step: 0, //starts at 0 but this is also causing elements to refresh at step 0, which is a no-no
-            progress: 0,
-
-            // force
-            force_sim: null,
-            init_decay: null,
-
-            timeBeforeKill: 5000,
-            exp_color: ["orangered", "teal"],
-
-           //listing model steps in order for the two separate datasets
-            model_list: ['ANN','ANN','ANN','ANN','ANN','ANN', 'ANN', 'ANN', 'RNN','RNN', 'RGCN', 'RGCN', 'RGCN_ptrn','RGCN_ptrn'],
-            model_list_cast: ['ANN_d001','ANN_d001','ANN_d001','ANN_d001','ANN_d001', 'ANN_d001', 'ANN_d001', 'RNN_d001','RNN_d001', 'RGCN_d001', 'RGCN_d001', 'RGCN_ptrn_d001','RGCN_ptrn_d001'],
+            step: 0, //starts at 0 but this is also causing elements to refresh at step 0
```

Comment:
if it would help on your end, I can easily set up flubber to not rely on this being set at 0 to start

Reply:
for the beeswarm I think it matter less BUT it could definitely be improved by binding the beeswarm data to the step in a different way that doesn't depend on the step index like it currently does